include/knowrob/py/wrapper.h

```diff
@@ -25,7 +26,7 @@ namespace knowrob::py {
 			return python::call_method<R>(self, method, python::object(args)...);
 		} catch (const python::error_already_set&) {
 			knowrob::py::handlePythonError();
-			throw ReasonerError("Python error");
+			throw ReasonerError("Python error", boost::stacktrace::stacktrace());
```

Comment:
the `handlePythonError` does throw an error already or? So this should never be reached I think.